pkgs/os-specific/linux/cpupower/gui.nix

```diff
@@ -0,0 +1,69 @@
+{ lib, stdenv, fetchFromGitHub
```

Comment:
can these all just be on their own line?

Reply:
done